vmdb/spec/models/ems_refresh/refreshers/vc_refresher_errors_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+
+describe EmsRefresh::Refreshers::VcRefresher do
+  before(:each) do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(
+      :ems_vmware,
+      :zone      => zone,
+      :hostname  => "1.2.3.4",
+      :ipaddress => "1.2.3.4",
+      :port      => 5000)
+  end
+
+  it "will record an error when trying to perform a full refresh against Vmware" do
+    refresh_ems(@ems)
+    assert_failed_refresh
+  end
+
+  def assert_failed_refresh
+    @ems.last_refresh_status.should == "error"
+  end
+
+  def refresh_ems(ems)
+    EmsRefresh::Refreshers::VcRefresher.stub(:get_vc_data).and_raise(Exception.new("Error getting VC Data"))
```

Comment:
Can we raise a different exception than "Exception"?  Maybe StandardError?  Technically your code would not catch this exception anyway.